src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/TensorsInfo.cs

```diff
@@ -469,19 +474,46 @@ protected virtual void Dispose(bool disposing)
             _disposed = true;
         }
 
+        private static int[] ConvertDimension(int[] dimension)
+        {
+            if (dimension == null) {
+                throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, "dimension is null");
```

Comment:
suggestion: update exception message "The dimension is null, it should be a valid array."

Reply:
Updated exception message!
Thank you